L1Trigger/L1THGCal/python/hgcalConcentratorProducer_cfi.py

```diff
@@ -177,6 +177,8 @@
                                  ctcSize = cms.vuint32(CTC_SIZE),
                                  modelFiles = cms.VPSet([autoEncoder_training_2eLinks, autoEncoder_training_3eLinks, autoEncoder_training_4eLinks, autoEncoder_training_5eLinks]),
                                  linkToGraphMap = cms.vuint32(linkToGraphMapping),
+                                 zeroSuppresionThreshold = cms.double(0.5),
```

Comment:
Is `0.5 mipT` the correct threshold? Or should it be `0.1 mipT`

Reply:
Sorry, guess I forgot to change that after my tests to find the optimal value.  I've updated this to be 0.1, as well as squashed things back to one commit